cmd/cli/kubectl-kyverno/commands/test/compare.go

```diff
@@ -35,6 +38,12 @@ func getAndCompareResource(actualResources []*unstructured.Unstructured, fs bill
 			return false, fmt.Errorf("error: failed to compare resources (%s)", err)
 		}
 		if !equals {
+			log.Log.V(8).Info("Resource diff", "expected", expectedResourcesMap[r.GetNamespace()+"/"+r.GetName()], "actual", r)
```

Comment:
Can we move these to the debug log to level 4, not sure if we use level 8 for anything.

Reply:
Yeah, sure! I'll patch to that effect.

I'm of a mind to try to actually expose it under `--detailed-results` (i.e. in the same output box as where it currently just says "there was a diff") but maybe that will have to wait for a separate PR.